src/main/java/seedu/address/logic/commands/ListCommand.java

```diff
@@ -12,13 +12,14 @@ public class ListCommand extends Command {
 
     public static final String COMMAND_WORD = "list";
 
-    public static final String MESSAGE_SUCCESS = "Listed all persons";
+    public static final String MESSAGE_SUCCESS = "Listed all persons \n%s";
 
     @Override
     public CommandResult execute(Model model) {
         requireNonNull(model);
         model.setListView();
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
```

Comment:
you're calling `updateFilteredPersonList` here and it gets called again immediately after in `getNextSession`, which is unneccessary. i think this one should be removed and only called in `getNextSession`

Reply:
Hmm this one is not from my change but if im not wrong, its ok if i remove the `updateFilteredPersonList` call in the `getNextSession` because:

1) `getNextSession` is called on launch of app where everyone is shown

2) `getNextSession` is called after list command is called, which means the `model.updateFilteredPersonList` is called as above

Validate my logic @phyliciachristel?

Reply:
i mean my point is that you're calling the same method twice in a row, doesn't matter which you remove since both methods will be called all the time

Reply:
hi its ok i decided instead to pass the personList to NextSessionUtil!